storage/execout/configs.go

```diff
@@ -20,10 +20,7 @@ func NewConfigs(baseObjectStore dstore.Store, allRequestedModules []*pbsubstream
 	out := make(map[string]*Config)
 	for _, mod := range allRequestedModules {
 
-		initialBlock := mod.InitialBlock
-		if initialBlock < firstStreamableBlock {
-			initialBlock = firstStreamableBlock
-		}
+		initialBlock := max(mod.InitialBlock, firstStreamableBlock)
```

Comment:
Let's revert `mod.InitialBlock` and `firstStreamableBlock` in the three cases
```suggestion
		initialBlock := max(firstStreamableBlock, mod.InitialBlock)
```

Reply:
reverted.